src/shared/utils/ast/nodes.ts

```diff
@@ -0,0 +1,38 @@
+import {get} from 'lodash'
```

Comment:
I think I would concur. Copying code is great, but I'm a little hesitant to knowingly copy tech debt/bugs. Do we have a preferred API instead of `_.get`? If it's a minimal lift, maybe this is an appropriate change, given the size of the patch.

Reply:
Extracting nested objects in an AST subtree, with some flexibility on the children nodes.
In the simplest case, this means (for example) replacing:
`return get(node, 'callee.name') === 'now'`
with:
`return node.callee && node.callee.hasOwnProperty('name') && (node.callee as any).name === 'now'`

Or similar patterns where we manually handle the uncertainty of the AST children subnodes, especially when it comes to typescript types.

So we can do that...if we are good with this tradeoff.

Reply:
I defer to @asalem1, who has more understanding of the history is here. I definitely am not a fan of the more verbose spelling, but I also get that we kinda don't have a great story for traversing/transforming AST in general.